test/caesar.test.js

```diff
@@ -0,0 +1,17 @@
+import caesar from '../js/caesar';
+
+test('correctly cipher text', () => {
+  expect(caesar.caesarCypher('hello', 2)).toBe('jgnnq');
+});
+
+test('Only letters can be included', () => {
+  expect(caesar.caesarCypher(3, 1)).toBe('The input has to be string');
+});
+
+test('Uppercase letters also works', () => {
+  expect(caesar.caesarCypher('HELLO', 2)).toBe('jgnnq');
+});
+
+test('Punctuation and special characters are not converted', () => {
+  expect(caesar.caesarCypher('h.ello', 2)).toBe('j.gnnq');
+});
```

Comment:
#### Caesar Cipher ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/caesar_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/testing-practice))

* [ ] It should have a test for wrapping from `z` to `a`
* [ ] It should have a test for keeping the same case
* [ ] It should have a test for punctuation